lib/elixir/lib/dynamic_supervisor.ex

```diff
@@ -78,20 +78,20 @@ defmodule DynamicSupervisor do
       defmodule MySupervisor do
         use Supervisor
 
-        def start_link(arg) do
-          Supervisor.start_link(__MODULE__, arg, name: __MODULE__)
+        def start_link(extra_arg) do
```

Comment:
Why  are we not using  `init_arg` here too? 🤔

Reply:
Check, I pushed an updated commit.